supervisor/filesystem.h

```diff
@@ -31,6 +31,8 @@
 
 void filesystem_init(bool create_allowed);
 void filesystem_flush(void);
+// Writeable indicates that the Python VM can write to the filesystem and USB
+// can't.
 void filesystem_default_writeable(bool writeable);
```

Comment:
Can we use a better name, since the meaning is unclear except due to the comment? `filesystem_default_writable_by_python` or `filesystem_default_python_writable`, or something similar?

Reply:
I renames this to `filesystem_writable_by_python`.